### ISSUE TITLE
Moscow Wifi Free

### ISSUE THREAD

**Issue by ogofano** [bol-van/zapret#694]
Приветствую. Возникла проблема, что при авторизации в сети Moscow Wifi free окно авторизации просто не загружается. Пробовал добавлять в исключения beelinewifi.com, но результата не дало. Что можно сделать?

**Comment by HentayHentay** [comment]
Возникло сразу пачку вопросов:
- А как-же требования root?
- Очень интересно для чего запрет в таком конфиге
- Запрет дорос до телефонов. Поздравляем всех.... Телефоны уже?
- Как отреагировала Москва Фри?

**Comment by ogofano** [comment]
Возможно я как-то неправильно добавлял в исключения, потому что проблема есть еще с парочкой сайтов..

**Comment by HentayHentay** [comment]
**Был обнаружен самоубийца...**

**Comment by bol-van** [comment]
Варианты разные есть.
Если стратегия только с TTL, то ломаются близкие сайты. Нужен доп ограничитель.
Исключения можно по IP или по хостам внести.
Отследить куда обращается в шарке. Какие IP, хосты. Дальше думать по IP или хостам исключать

**Comment by ogofano** [comment]
А можно инструкции по-подробнее, так сказать для ламеров в этой теме. Мне бы, чтобы ютуб и дискорд работали) Остальное не суть важно

**Comment by HentayHentay** [comment]
> А можно инструкции по-подробнее, так сказать для ламеров в этой теме. Мне бы, чтобы ютуб и дискорд работали) Остальное не суть важно

[readme.txt](https://github.com/bol-van/zapret/blob/master/docs/readme.txt)
[quick_start.txt](https://github.com/bol-van/zapret/blob/master/docs/quick_start.txt)

**Comment by ogofano** [comment]
Эти файлы доступны всем. Вы думаете, что я бы стал писать вопрос, если бы итак мог решить проблему самостоятельно?

> > А можно инструкции по-подробнее, так сказать для ламеров в этой теме. Мне бы, чтобы ютуб и дискорд работали) Остальное не суть важно
> 
> [readme.txt](https://github.com/bol-van/zapret/blob/master/docs/readme.txt) <- подробная [quick_start.txt](https://github.com/bol-van/zapret/blob/master/docs/quick_start.txt) <- **я не хочу разбираться**

**Comment by HentayHentay** [comment]
> Эти файлы доступны всем. Вы думаете, что я бы стал писать вопрос, если бы итак мог решить проблему самостоятельно?

Запрет рассчитан на самостоятельный поиск в силу того, что у каждого провайдера свои трудности, проблемы и ограничения.  Стратегии обхода и советы от других часто не подходят людям, которые спрашивают. А сборки, которые есть имеют и так те или иные проблемы.

**Comment by Fenrir-0xFF** [comment]
> А можно инструкции по-подробнее, так сказать для ламеров в этой теме. Мне бы, чтобы ютуб и дискорд работали) Остальное не суть важно

На следующей недели обновлённый скрипт для nfqws сюда выложу, там без всяких заморочек всего пара команд start и stop. Делал его ещё давно для андроида и openwrt, скоро буду переделывать с учётом возможностей использования мульти стратегий. 

Касательно авторизации в публичных вайфаях - она осуществляется по http а не по https, хотя не исключяется вероятность что может редиректнуть с http страницы на https. Поумолчянию страница любойДомен:80/gen_204* или ^generate_204* 
В конфиге http должно быть --dpi-desync-fooling=md5sig,badsum или вобще отключи http, т.к. сайты его уже практически не используют

**Comment by ogofano** [comment]
> Зайди к примеру на http://ya.ru/gen_204 с публичного вайфая

Если попробовать зайти, то перекидывает на https и выдает 404 ошибка, нет такой страницы

**Comment by Fenrir-0xFF** [comment]
> > Зайди к примеру на http://ya.ru/gen_204 с публичного вайфая
> 
> Если попробовать зайти, то перекидывает на https и выдает 404 ошибка, нет такой страницы

Ошибка яндексовская или WiFi веб сервера? Если 404я ошибка то скорей всего у тебя интернет итак работает, потомучто любые сайты с любыми get запросами по http должно перенаправлять на страницу авторизации если она не была пройдена ранее. 
Без /gen_204 любой сайт по http открыть попробуй.

**Comment by ogofano** [comment]
> > > Зайди к примеру на http://ya.ru/gen_204 с публичного вайфая
> > 
> > 
> > Если попробовать зайти, то перекидывает на https и выдает 404 ошибка, нет такой страницы
> 
> Ошибка яндексовская или WiFi веб сервера? Если 404я ошибка то скорей всего у тебя интернет итак работает, потомучто любые сайты с любыми get запросами по http должно перенаправлять на страницу авторизации если она не была пройдена ранее. Без /gen_204 любой сайт по http открыть попробуй.

Ошибка яндекса. Если на любой другой сайт попробовать, то сразу на https кидает

**Comment by Fenrir-0xFF** [comment]
> Ошибка яндекса. Если на любой другой сайт попробовать, то сразу на https кидает

Если у тебя интернет работает то какого хрена людям мозги заморачиваеш? 
Окно авторизации вылазит только если эта авторизация не пройдена, а у тебя она была пройдена ранее. 
Если так хочется пройти её снова то смени mac адрес на другой.

**Comment by HentayHentay** [comment]
@Fenrir-0xFF 
Он хочет через бесплатные сети  получить информацию, которая в бане по стране (аля инстраграм или мордокнига)

**Comment by Fenrir-0xFF** [comment]
> @Fenrir-0xFF Он хочет через бесплатные сети получить информацию, которая в бане по стране (аля инстраграм или мордокнига)

Насколько понял проблема у него была в том что не открывалось окно авторизации публичного вайфая, а не открывалось оно потомучто оказывается он там уже авторизовался. Вобшем пофигу. 

Я гдето через чясик скрипт выложу, там упрощённый запуск nfqws без лишних заморочек, пускай чел тогда его протестит, предыдущяя версия скрипта на публичных вайфаях у меня робила. 
Хотел только на неделе выложить чтобы народ сначяла на 4pda о багах на официальных (урезанных) прошивках сообщил, но видимо придётся сёдня.

**Comment by ogofano** [comment]
> > Ошибка яндекса. Если на любой другой сайт попробовать, то сразу на https кидает
> 
> Если у тебя интернет работает то какого хрена людям мозги заморачиваеш? Окно авторизации вылазит только если эта авторизация не пройдена, а у тебя она была пройдена ранее. Если так хочется пройти её снова то смени mac адрес на другой.

Мы друг друга не поняли. Чтобы авторизироваться я отключал запрет через uninstall, а после его включал. Так вроде все работает, за исключением пары сайтов. И я похоже неправильно добавляю их в исключения, т.к. все равно они не открываются после. А еще после авторизации интернет работает, а сеть вот таким странным образом отображается
![image](https://github.com/user-attachments/assets/0acbd170-50d5-4dcc-9e3c-f66ea6c5a62a)

**Comment by ogofano** [comment]
> Насколько понял проблема у него была в том что не открывалось окно авторизации публичного вайфая, а не открывалось оно потомучто оказывается он там уже авторизовался.

Нет, оно не открывается по каким-то другим, непонятным мне причинам. И приходится авторизироваться, временно выключая запрет

**Comment by HentayHentay** [comment]
А может там активно ТПСУ работает, а ты под запретом в открытых сетях ходишь. Как итог - ты ловишь ограничение сети. 
Если не "гений" - прекрати быть собакой Павлова и не тренируй DPI
